go.mod

```diff
@@ -5,7 +5,7 @@ go 1.14
 require (
 	github.com/gdamore/tcell v1.3.0
 	github.com/hajimehoshi/go-mp3 v0.3.0
-	github.com/hajimehoshi/oto v0.7.1
+	github.com/hajimehoshi/oto/v2 v2.4.0-alpha.4
```

Comment:
The alpha version contains a fix for [this locking problem](https://github.com/hajimehoshi/oto/issues/188)

Reply:
alpha.5 was released very recently.